.github/labeler-file-extensions.yml

```diff
@@ -0,0 +1,8 @@
+cython:
+- sklearn/**/*.pyx
+- sklearn/**/*.pxd
+- sklearn/**/*.pxi
+# Tapita templates
```

Comment:
```suggestion
# Tempita templates
```

Also nit: `sklearn/**/*.tp` would probably be enough: we never use tempita for non-Cython related thing